### ISSUE TITLE
[BUG]: About me tab section missing correct markup

### ISSUE THREAD

**Issue by emmalearnscode** [JayantGoel001/JayantGoel001.github.io#19]
### Description of the bug

The tab section in About me is not keyboard accessible and is missing the correct markup to tell screen reader users that these are tabs. 

Here's an example: https://www.w3.org/WAI/ARIA/apg/example-index/tabs/tabs-automatic.html

### Steps To Reproduce

Try to tab to the about me section
